Automattic/wp-calypso
# Hosting Command Palette: Replace arrow down with the download icon for importing site to WordPress command

Related to https://github.com/Automattic/wp-calypso/issues/84870

## Proposed Changes

*  This PR replaces the icon for the `Import site to WordPress.com` command and changes it from `arrowDown` to `download` icon as suggested https://github.com/Automattic/wp-calypso/issues/84870#issuecomment-1840931284

## Testing Instructions

* Navigate to Calypso Live Link
* Press `cmd + k` on macOS or `ctrl + k` on Windows to open the command palette
* Search for the `Import site to WordPress.com` command
* Confirm that the icon for the command is `download`

<img width="1281" alt="Screenshot 2023-12-11 at 12 06 04 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/59bd8603-bbe5-4fde-b3a6-17d529fb1dbd">

## Pre-merge Checklist
